### ISSUE TITLE
[ISSUE] Unable to configure "users" and "admins" groups

### ISSUE THREAD

**Issue by pietern** [databrickslabs/terraform-provider-databricks#1150]
This is caused by #1091.

### Configuration

```hcl
resource "databricks_group" "users" {
  display_name          = "users"
  allow_cluster_create  = true
  databricks_sql_access = true
  workspace_access      = true

  force = true # implicitly import this pre-defined group
}
```

### Expected Behavior

Update the users group to give all users cluster creation capability.

### Actual Behavior

With the change in #1091 it's no longer possible to update this pre-existing group.

Possible solution is to make create and destroy noops and allow read and update to go through.

**Comment by pietern** [comment]
We have used these to make sure all users have cluster create capability. Without this it needs to be manually configured.

**Comment by nfx** [comment]
> We have used these to make sure all users have cluster create capability. Without this it needs to be manually configured.

for entitlements?

**Comment by pietern** [comment]
Yes. I realize this is a special case because you can't create or delete them, but they are still configurable.

**Comment by nfx** [comment]
> Yes. I realize this is a special case because you can't create or delete them, but they are still configurable.

Terraform's semantics is to use "data" (read-only, not configure) resources for those - https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/data-sources/group. 

> Without this it needs to be manually configured.

you can achieve the same by cluster policy and permissions.

```hcl
resource "databricks_cluster_policy" "something_simple" {
  name = "Some simple policy"
  definition = jsonencode({
    "spark_conf.spark.hadoop.javax.jdo.option.ConnectionURL" : {
      "type" : "forbidden"
    },
    "spark_conf.spark.secondkey" : {
      "type" : "forbidden"
    }
  })
}

resource "databricks_permissions" "policy_usage" {
  cluster_policy_id = databricks_cluster_policy.something_simple.id

  access_control {
    group_name       = "users"
    permission_level = "CAN_USE"
  }
}
```
